Makefile

```diff
@@ -12,8 +12,8 @@ BEEKEEPER_BRANCH ?= master
 REACHABILITY_OVERRIDE_PUBLIC ?= false
 BATCHFACTOR_OVERRIDE_PUBLIC ?= 5
 
-GO_MIN_VERSION ?= "1.17"
-GO_BUILD_VERSION ?= "1.17.2"
+GO_MIN_VERSION ?= "1.18"
+GO_BUILD_VERSION ?= "1.18.1"
```

Comment:
it seems that the next minor version has been released meanwhile, not sure it's big enough of a deal

https://github.com/golang/go/releases/tag/go1.18.2

Reply:
Thanks, upgraded!